journals/apps/journals/models.py

```diff
@@ -401,6 +406,32 @@ def root_journal_page_url(self):
         descendants = self.get_descendants()
         return descendants[0].full_url if descendants else '#'
 
+    def get_journal_structure(self):
```

Comment:
probably would be best to not duplicate this method as it already exists on JournalPage. Perhaps it could be abstracted out and used by both pages?

Reply:
or another option: The JournalAboutPage has an associated Journal. If you find the JournalPage for the Journal then you can call get_journal_structure directly on the JournalPage and then the method doesn't need to be refactored.

Reply:
Actually I was thinking it might make sense to have a class called ```JournalPageMixin``` which could have methods that apply to both ```JournalAboutPage``` and ```JournalPage```.   Right now it would just be ```get_journal_structure```, but there is a good chance we will have more shared methods in the future